build.sbt

```diff
@@ -59,7 +55,9 @@ val ignoredABIProblems = {
     exclude[MissingMethodProblem]("com.twitter.storehaus.asynchbase.AsyncHBaseStringStore" +
       ".futurePool"),
     exclude[MissingMethodProblem]("com.twitter.storehaus.asynchbase.AsyncHBaseStore.futurePool"),
-    exclude[MissingMethodProblem]("com.twitter.storehaus.ReadThroughStore.mutex")
+    exclude[MissingMethodProblem]("com.twitter.storehaus.ReadThroughStore.mutex"),
+    exclude[MissingClassProblem]("com.twitter.storehaus.kafka.JavaFutureToTwitterFutureConverter$Closed$"),
```

Comment:
I think this code got migrated to bijection anyway, so we can use that on the next bump.